docs/best-practices/feature-flag-cleanup.md

```diff
@@ -31,13 +31,23 @@ Practicing healthy feature flag hygiene is a good way to mitigate technical debt
 
 You should also remove a flag or its variables if they are inactive or not in use. Perhaps the feature was never shipped, or it was shipped with a different flag or variable. Either way, removing it will clear the clutter in your dashboard and your code.
 
+## Using DevCycle CLI to cleanup feature flags
+
+Currently only supported in JavaScript.
```

Comment:
We could explicitly say that it's supported for the javascript, node, and react SDKs

Reply:
do we want to say sdk's or mention file types

Reply:
In the context of our docs it would probably make the most sense to mention the SDK types (since the file types are then implied). If we want to explicitly mention the file types it'd be js, ts, jsx and tsx (to include typescript and react files)